requirements.txt

```diff
@@ -66,7 +66,7 @@ statsd==3.2.1
 transaction==2.1.2
 translationstring==1.3    # via colander, deform, pyramid
 unidecode==0.4.19         # via python-slugify
-urllib3==1.24.1             # via elasticsearch
```

Comment:
I'm not sure why this wasn't here before? But I don't think this is a big deal either way.